lib/export.js

```diff
@@ -3513,14 +3519,26 @@ class FHIRExporter {
     }
   }
 
-  lookupProfile(identifier, createIfNeeded=true, warnIfProfileIsProcessing=false) {
+  lookupProfile(identifier, createIfNeeded=true, warnIfProfileIsProcessing=false, warnIfUsingMappingNoProfile=false) {
     const mapping = this._specs.maps.findByTargetAndIdentifier(this._target, identifier);
+    // Maintain support for 'no profile' in mapping file
     if (mapping != null) {
       const targetItem = common.TargetItem.parse(mapping.targetItem);
       if (targetItem.hasNoProfileCommand()) {
+        // For now allow setting no profile in mapping file, but warn that it should be stopped
+        if (warnIfUsingMappingNoProfile) {
+          logger.warn("Setting 'no profile' for %s in mapping file is deprecated and should be set in content profile. ERROR_CODE:03005", 
```

Comment:
Could you edit this wording to say:
`"Setting 'no profile' for %s in mapping file is deprecated and should be done in content profile instead. ERROR_CODE:03005"`